scripts/gen_conda_env_file.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 
 # NB: This script should be run from package folder containing pyproject.toml
+# NB: package myia_utils must be installed.
 
 # Install python dependencies required to run this script.
 pip install poetry2conda==0.3.0
 
 # Generate environment.yml from pyproject using poetry2conda
 poetry2conda pyproject.toml --dev -E pytorch environment.yml
+
+# Remove name field from environment file.
+python -m myia_utils.update_env environment.yml -o environment.yml
```

Comment:
Why?

Reply:
If name is in env yml file, then conda will try to create an environment with this name and install packages in it. It was also discussed here: https://github.com/mila-iqia/myia/pull/369#discussion_r470190420

So, to avoid any future issues, I must remove name field from any environment yml file. I did it for other packages (myia_backend_relay and myia_backend_pytorch) in previous PR, but I forgot to do that for main myia package here!